Branch-SDK/src/main/java/io/branch/referral/PrefHelper.java

```diff
@@ -266,6 +270,48 @@ public void setTimeout(int timeout) {
     public int getTimeout() {
         return getInteger(KEY_TIMEOUT, TIMEOUT);
     }
+
+    /**
+     * <p>Sets the duration in milliseconds to override the timeout value for tasks</p>
+     *
+     * @param taskTimeout The {@link Integer} value of the timeout setting in milliseconds.
+     */
+    public void setTaskTimeout(int taskTimeout) {
```

Comment:
I would drop this setter and have the getter return a computed value.

Reply:
Done